tensordict/tensordict.py

```diff
@@ -2021,9 +2021,10 @@ def _index_tensordict(self, idx: INDEX_TYPING):
     def __getitem__(self, idx: INDEX_TYPING) -> TensorDictBase:
         """Indexes all tensors according to the provided index.
 
-        Returns a new tensordict where the values share the storage of the original tensors (even
-        when the index is a torch.Tensor). Any in-place modification to the
-        resulting tensordict will impact the parent tensordict too.
+        Returns a new tensordict where the values share the storage of the
+        original tensors (even when the index is a torch.Tensor).
```

Comment:
Actually reading this makes me think it is not true for `torch.Tensor` indices. I think now you need to call `get_subtensordict` explicitely to get a tensor that shares the same data pointer.

Reply:
It's a "doctring issue" or code one? I haven't changed __getitem__. Just followed linter warning for the docstring

Reply:
it's a docstring issue, more a note to myself than a comment on this PR

Reply:
Opened #240